default/python/AI/FreeOrionAI.py

```diff
@@ -3,12 +3,16 @@
 import pickle  # Python object serialization library
 import sys
 import random
+import logging
 
-from common.configure_logging import redirect_logging_to_freeorion_logger
-redirect_logging_to_freeorion_logger()
+from common.configure_logging import redirect_logging_to_freeorion_logger, convenience_function_references_for_logger
 
 import freeOrionAIInterface as fo  # interface used to interact with FreeOrion AI client  # pylint: disable=import-error
 
+log_level = logging.DEBUG if fo.getOptionsDBOptionBool("verbose-logging") else logging.INFO
```

Comment:
I did not wire up `--log-level`, just `--verbose-logging` and I only checked that it worked from `persistent_config.xml`  not the command line.  I'll look into that.

I don't want to get too deep into wiring up the back end until you finish the log rollover code.  This PR was planned to be just the python portion and controlled from within the python code itself.